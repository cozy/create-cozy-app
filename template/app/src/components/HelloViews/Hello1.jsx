import React from 'react'

export const Hello1 = () => (
  <p>Just... Hello world! This is a first hello view</p>
)

export default Hello1
