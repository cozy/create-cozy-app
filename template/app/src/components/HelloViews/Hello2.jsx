import React from 'react'

export const Hello2 = () => (
  <p>Just... Hello world! This is a second hello view</p>
)

export default Hello2
