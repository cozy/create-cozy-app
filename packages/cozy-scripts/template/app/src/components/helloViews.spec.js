'use strict'

/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'

import { Hello1 } from 'components/HelloViews/Hello1'
import { Hello2 } from 'components/HelloViews/Hello2'

describe('Hello components:', () => {
  it('Hello1 should be rendered correctly', () => {
    const component = shallow(<Hello1 />).getElement()
    expect(component).toMatchSnapshot()
  })
  it('Hello2 should be rendered correctly', () => {
    const component = shallow(<Hello2 />).getElement()
    expect(component).toMatchSnapshot()
  })
})
