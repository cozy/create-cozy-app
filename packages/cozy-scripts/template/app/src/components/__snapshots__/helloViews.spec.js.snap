// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello components: Hello1 should be rendered correctly 1`] = `
<p>
  Just... Hello world! This is a first hello view
</p>
`;

exports[`Hello components: Hello2 should be rendered correctly 1`] = `
<p>
  Just... Hello world! This is a second hello view
</p>
`;
